Anna_Chung.cfdg

```diff
@@ -0,0 +1,43 @@
+startshape Peacock
```

Comment:
Super cool art! I like how you exploited some of the strengths of ContextFree while also creating a recognizable peacock.